todo-angular-koa/app.js

```diff
@@ -84,7 +84,7 @@ function* update(next) {
             throw new Error("The todo must have a field `id`.");
         }
 
-        var result = yield r.table('todos').get(todo.id).update(todo, {returnVals: true}).run(this._rdbConn);
+        var result = yield r.table('todos').get(todo.id).update(todo, {returnChanges: true}).run(this._rdbConn);
         this.body = JSON.stringify(result.new_val);
```

Comment:
You are right! Didn't think about it enough. I opened https://github.com/rethinkdb/rethinkdb-example-nodejs/issues/14 so we don't forget changing that.